css/snake.css

```diff
@@ -0,0 +1,15 @@
+#snake-canvas {
+  position: absolute;
+}
+
+#snake-canvas-container {
+  position: relative;
+  width: 400px;
+  height: 400px;
+}
+
+#snake-game-over {
```

Comment:
Figure out if there's a cleaner way to do this instead of using Absolute positioning

Reply:
Wasn't able to find a way to avoid this without it breaking at different screen sizess